docs/quickstart.rst

```diff
@@ -0,0 +1,91 @@
+Quickstart
+==========
+
+Installation
+------------
+
+Install the extension:
+
+.. code-block:: console
+
+   $ pip install sphinx-multiproject
+
+Usage
+-----
+
+You can structure your project like this:
+
+.. code-block::
+
+   docs
+   ├── conf.py
+   ├── dev
+   │   ├── conf.py
+   │   └── index.rst
+   └── user
+       ├── conf.py
+       └── index.rst
+
+Where ``docs/conf.py`` would be your shared configuration file,
+and ``docs/dev/`` and ``docs/user/`` would be your projects
+using their specific configuration file.
+
+.. code-block:: python
+
+   # File: docs/conf.py
+
+   extensions = [
+      "multiproject.extension",
```

Comment:
Perhaps this is a pattern to reign in. We did this historically on a few extensions, but most modern Sphinx extension just use the top level package name:

```suggestion
      "multiproject",
```

This is annoying to change later, so might be worth doing now, unless there is a particular reason we need the nested namespace.

Reply:
This was so it wont give an import error at https://github.com/readthedocs/sphinx-multiproject/blob/b82ae09ecda055148ae9be13b6326534a6e75a95/multiproject/__init__.py#L1-L1 when installing the extension (since sphinx isn't installed yet). But looks like when not using a `setup.py` (but a setup.cfg) the file isn't evaluated

https://github.com/readthedocs/sphinx-multiproject/blob/b82ae09ecda055148ae9be13b6326534a6e75a95/setup.cfg#L3-L3

Did a quick test and looks like shouldn't be a problem :)

Reply:
Oh nice, I didn't know that was possible!

I've been liking bump2version for this as well, and avoiding all of the import issues on `__version__` entirely (plus versions strings like in package.json/etc)